components/docs/DocsCards.js

```diff
@@ -37,13 +38,20 @@ const cards = [
     className: 'bg-primary-200',
     Icon: IconGuide,
   },
+  {
+    title: 'Community',
+    description: 'Our Community section documents the community guidelines and resources.',
```

Comment:
```suggestion
    description: 'Our Community section documents **community** _processes_, **contribution guidelines**, and **resources**.',
```

Reply:
how can we add this styling in this js file? 😄😬😬😬😬

Reply:
We can, but what about other buckets and theirs description? Should we also bold some parts?

Reply:
yeah good point... maybe we shouldn't 🧐